plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteContainer.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ignite;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+
+public class TestIgniteContainer
+        extends JdbcDatabaseContainer
+{
+    public static final String NAME = "ignite";
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("apacheignite/ignite");
+    /**
+     * @deprecated
+     */
+    @Deprecated
+    public static final String IMAGE = DEFAULT_IMAGE_NAME.getUnversionedPart();
+
+    public static final String DEFAULT_TAG = "2.10.0";
+    public static final Integer HTTP_PORT = 10800;
+    public static final Integer NATIVE_PORT = 18000;
```

Comment:
Why the aboves are public?

Reply:
HTTP_PORT is used in other class, other properties are changed to ``private``